docs/guides/developer/docs/development-process.md

```diff
@@ -86,6 +86,21 @@ modules/admin-ui/src/main/java | In case the interface of the Admin UI facade ch
 While a committer may accept a patch even if it does not meet these expectations, it is encouraged that anyone filing
 a pull request ensures that they meet these expectations.
 
+#### Merging Pull Requests
+
+After a pull request has received at least one approving review and passes the automated tests, it is ready for merging.
+Only a committer can perform a merge, so if a reviewed pull request has not yet received attention from a committer
+feel free to contact one.
```

Comment:
Who's `me`?

Reply:
Are you suggesting to put a link to the list of current committers there?

Reply:
No, I'm asking who that sentence is referring to - me or Lars I'm guessing, but someone not as familiar with the community wouldn't have a clue.